src/M6Web/Bundle/LogBridgeBundle/Config/FilterParser.php

```diff
@@ -59,11 +67,15 @@ protected function isRoute(string $name): bool
     public function parse(string $name, array $config): Filter
     {
         if (
-            !array_key_exists('route', $config) ||
+            (!array_key_exists('route', $config) && !array_key_exists('routes', $config)) ||
             !array_key_exists('method', $config) ||
             !array_key_exists('status', $config)
         ) {
-            throw new ParseException(sprintf('Undefined "route", "method" or "status" parameter from filter "%s"', $name));
+            throw new ParseException(sprintf('Undefined "route(s)", "method" or "status" parameter from filter "%s"', $name));
+        }
+
+        if (array_key_exists('route', $config) && array_key_exists('routes', $config)) {
+            throw new ParseException(sprintf('You can\'t use both "route" and "routes" parameter from filter "%s"', $name));
```

Comment:
I was not expected this. instead of create a 2 if condition for route(s) config you can use the xor operator.
so in the first condition you can replace `(!array_key_exists('route', $config) && !array_key_exists('routes', $config))` by `(!(array_key_exists('route', $config) xor array_key_exists('routes', $config)))`

Reply:
We saw this together like a family, it's ok to have both if